solutions/classes/flight.js

```diff
@@ -0,0 +1,29 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+	idx++;
+	return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+const Flight = class{
```

Comment:
@Rahulkug Please use `Class Declaration` instead of `Class Expression` ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes )

Reply:
@sailesh-ramayanam  made changes.

Reply:
@sailesh-ramayanam made changes in the code.